build/__main__.py

```diff
@@ -80,6 +80,9 @@ def main(cli_args):  # type: (List[str]) -> None
     parser.add_argument('--skip-dependencies', '-x',
                         action='store_true',
                         help='does not check for the dependencies')
+    parser.add_argument('--no-isolation', '-n',
+                        action='store_true',
+                        help='do not isolate the build in a virtual environment')
```

Comment:
would be worthwhile to mention that will raise error if build dependencies are not specified; ``do not create an isolated build environment (raise error when host is missing build requirements)`` 🤔

Reply:
I would go with "requires build dependencies to be present". What do you think?

Reply:
Works for me 👍